source/api/commands/invoke.md

```diff
@@ -194,6 +194,30 @@ cy
   .invoke('val').should('match', /apples/)
 ```
 
+### Retries
+
+The command `cy.invoke` automatically retries invoking the specified method, until any the returned value satisfied the attached assertions. The example below passes after 1 second.
```

Comment:
`cy.invoke` is invalid Cypress code, if it's a child command always refer as `.invoke()`

```suggestion
`.invoke()` automatically retries invoking the specified method until the returned value satisfies the attached assertions. The example below passes after 1 second.
```